src/js/letters/components/LetterList.jsx

```diff
@@ -14,21 +14,39 @@ class LetterList extends React.Component {
       let content;
 
       if (letter.letterType === 'benefit_summary') {
-        content = (<VeteranBenefitSummaryLetter/>);
+        content = (
+          <div>
+            <VeteranBenefitSummaryLetter/>
+            <DownloadLetterLink
+                letterType={letter.letterType}
+                letterName={letter.name}
+                downloadStatus={downloadStatus[letter.letterType]}
+                key={`download-link-${index}`}/>
+            <p>
+              If your service period or disability status information is incorrect, please send us
+              a message through VA’s <a target="_blank" href="https://iris.custhelp.com/app/ask">
+              Inquiry Routing & Information System (IRIS)</a>. VA will respond within 5 business days.
+            </p>
+          </div>
+        );
       } else {
-        content = letterContent[letter.letterType] || '';
+        content = (
+          <div>
+            <div>{letterContent[letter.letterType] || ''}</div>
+            <DownloadLetterLink
+                letterType={letter.letterType}
+                letterName={letter.name}
+                downloadStatus={downloadStatus[letter.letterType]}
+                key={`download-link-${index}`}/>
+          </div>
+        );
       }
 
       return (
         <CollapsiblePanel
             panelName={letter.name}
             key={`collapsiblePanel-${index}`}>
           <div>{content}</div>
-          <DownloadLetterLink
```

Comment:
To avoid duplicating the DownloadLetterLink code, what if we left all of the above as-is, and then just before the `return` statement for this `map` callback, define something like
```
let bslHelpInstructions;
if (letter.letterType === 'benefit_summary) {
   bslHelpnstructions = (<p>If your service period or disability status information is incorrect, please send us a message through VA’s <a target="_blank" href="https://iris.custhelp.com/app/ask"> Inquiry Routing & Information System (IRIS)</a>. VA will respond within 5 business days.</p>);
}
```
and then after the download link add
```
{bslHelpInstructions}
```

Reply:
That's so much better! Will do that, thanks!